planewatch/Dockerfile.template

```diff
@@ -0,0 +1,95 @@
+FROM balenalib/%%BALENA_ARCH%%-debian-golang:bookworm AS buildstep
+LABEL maintainer="https://github.com/ketilmo"
+
+# specify plane.watch pw-feeder version
+# renovate: datasource=github-tags depName=plane-watch/pw-feeder versioning=loose
+ARG PLANEWATCH_VERSION=v0.0.4
+
+# specify mlat-client version
+ARG MLAT_VERSION=fe70767be859100176983b948140046b6ecdd34a
+
+# install build step pre-requs
+ARG TEMP_INSTALL="python3-dev python3-setuptools python3-venv"
+WORKDIR /tmp
+RUN apt update && \
+	apt install -y $TEMP_INSTALL
+
+# build pw-feeder binary from source
+WORKDIR /tmp
+RUN git clone --branch "$PLANEWATCH_VERSION" https://github.com/plane-watch/pw-feeder.git /tmp/pw-feeder && \
+	cd /tmp/pw-feeder/pw-feeder && \
+	go mod tidy && \
+    go generate -v ./... && \
+    go build -v ./cmd/pw-feeder/ && \
```

Comment:
Can't we use the prebuilt binaries?

Reply:
We could indeed. We make the feeder binary available here: https://github.com/plane-watch/pw-feeder/releases/.
If you'd prefer me to change the dockerfile to use pre-built binaries, please let me know. Building from source was easier that downloading an architecture-specific binary.

Reply:
In some of the other docker images, it uses the prebuilt one, which speeds things up. I'm okay either way, but just an idea.

Reply:
I'd prefer to build from source if that's ok.